packages/server/src/bandit/banditData.ts

```diff
@@ -0,0 +1,133 @@
+import { isProd } from '../lib/env';
+import * as AWS from 'aws-sdk';
```

Comment:
This is a good question, and something I've not given a lot of thought to in the past.
I think the answer is that it doesn't really matter in JS -

- It doesn't pollute the scope because you still have to reference the alias, e.g. `AWS.DynamoDB`.
- This project uses webpack for the server so I _think_ unused code gets stripped out during transpilation.

However, we could instead use:
`import { DynamoDB } from 'aws-sdk';`
Which I think is nicer!

Reply:
let's raise this as a Trello ticket (or possibly a github issue?) as it affects a lot of the codebase

Reply:
https://trello.com/c/pukrFrs0/153-sdc-importing-everything-from-aws-zod